GearWorksXO-2017/src/org/usfirst/frc/team2212/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,50 @@
+package org.usfirst.frc.team2212.robot.subsystems;
+
+import com.spikes2212.genericsubsystems.drivetrains.TankDrivetrain;
+
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.PIDSource;
+import edu.wpi.first.wpilibj.SpeedController;
+
+/**
+ *
+ */
+public class Drivetrain extends TankDrivetrain {
+
+	private SpeedController leftGearbox;
+	private SpeedController rightGearbox;
+	private Encoder encoderLeft;
+	private Encoder encoderRight;
+

```

Comment:
no need for the word gearbox here. we are not ... מכניקאים... YAK!
use leftSpeedController instead